ci/deps/azure-37-numpydev.yaml

```diff
@@ -3,20 +3,19 @@ channels:
   - defaults
 dependencies:
   - python=3.7.*
-  - pytz
-  - Cython>=0.29.13
-  # universal
-  # pytest < 5 until defaults has pytest-xdist>=1.29.0
-  - pytest>=4.0.2,<5.0
-  - pytest-xdist
+  # tools
+  - cython=0.29.13
```

Comment:
same (and all others except for the 3.6-min build can have >= for cython)

Reply:
In the minimum versions build we have a `>=0.29.13` now. There is a `=0.29.13` in another build, not sure if that was intentional.

I guess if we pin cython, when this causes a failure, the immediate action will be to pin it to something bigger where it works. So, is it really worth forcing in a build the `0.29.3` version?